static.config.js

```diff
@@ -295,14 +297,22 @@ const makeDepartmentPages = async client => {
   // copying the pattern from topics, may not need to do all this copying
   for (var infoPage of informationPages) {
     if (!infoPage.department) continue;
+
     infoPage.type = 'info';
 
     let matchingDepartmentIndex = departments.findIndex(
       d => d.id === infoPage.department.id,
     );
 
     if (departments[matchingDepartmentIndex]) {
-      departments[matchingDepartmentIndex].relatedLinks.push(infoPage);
+      // Only add the related link if it's not a top service
```

Comment:
this conditional block (308-315) seems almost identical to the one (341-346), can we make it more clear what is going on here?

Reply:
This one is for information pages, the other is for service pages, they end up as different objects here